api/v1alpha1/secretagentconfiguration_types.go

```diff
@@ -144,7 +144,6 @@ type SecretConfig struct {
 	Namespace string `json:"-" yaml:"namespace,omitempty"`
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:MinItems=1
-	// +kubebuilder:validation:UniqueItems=true
```

Comment:
Since we can't use this, we should add custom validation to ensure the keys are unique. Maybe a separate PR though.

Reply:
https://github.com/ForgeRock/secret-agent/issues/29